pkg/util/parallel/worker.go

```diff
@@ -16,7 +16,7 @@ func (w *bufWorker) TaskResult(err error) interface{} {
 	taskResult := &bufWorkerTaskResult{
 		buf:  w.buf,
 		err:  err,
-		data: []byte(w.buf.String()),
+		data: w.buf.Bytes(),
```

Comment:
It is the breaking change. The `Bytes()` method returns the slice that will be changed after the next buffer modification.

https://golang.org/pkg/bytes/#Buffer.Bytes
```
Bytes returns a slice of length b.Len() holding the unread portion of the buffer. The slice is valid for use only until the next buffer modification (that is, only until the next call to a method like Read, Write, Reset, or Truncate). The slice aliases the buffer content at least until the next buffer modification, so immediate changes to the slice will affect the result of future reads.
```

Reply:
Same as in comment above.